src/services/users.js

```diff
@@ -22,7 +23,9 @@ const createNewUser = async (user) => {
 	// eslint-disable-next-line max-len
 	const { rows, error } = await db.query(
```

Comment:
this isn't a change from this pr but kindly take out the `error` part along with the `if(error) throw error` statement later. It doesn't work that way and was merged mistakenly

Any error that happens will be thrown on its own without that

Reply:
yes please